### ISSUE TITLE
[Accessibility Audit]: Webinars

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5466]
This audit is a buddy task, to be preformed soon after the standardization of the Webinar pages.

### Webinars
- `/webinars/code-insights-turning-your-metrics-into-action`
- `/webinars/code-insights-turning-your-metrics-into-action?watch-now`
- `/webinars/preparing-for-the-next-log4j`
- `/webinars/preparing-for-the-next-log4j?watch-now`

### User Journey:
Since our webinars all use the same layout, auditing one webinar page should be sufficient. A cursory look over the other webinar pages can be given to ensure all behavior is the same.

- Navigate to a webinar page.
- Click through the watch video form.
- Be able to click any nav or footer link shown.
- Click through to play or pause the video shown upon form submission.

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

Video example [here](https://www.loom.com/share/21a5d671fc2a44b6be9f61335389c456).

1. Navigate user journey using **only** the keyboard
2. Navigate user journey **without** looking at your screen, using the screen reader.
3. Navigate user journey using a viewport with a **width of 320px**.
4. Work through relevant sections from the detailed checklist and ensure there are no issues.
5. If the issue is trivial & you have time, feel free to address the bug now. Otherwise, raise any larger bugs using the [accessibility issue template](https://github.com/sourcegraph/about/issues/new?assignees=&labels=accessibility%2Cwcag%2F2.1%2Cwcag%2F2.1%2Ffixing&template=accessibility_issue.yaml&title=%5BAccessibility%5D%3A+).

**Note:** [The Front End Platform team](https://join.slack.com/share/enQtMzY1OTY3MDkxNTAwOS03ZmFlNzgwNWJhOGU3NjU4MDY1NzVkOTdiNjQwMWQyNjJmNjBkNzk1OWQ5ZDIxOTNjNmRhOWIyMzUwMzhmZjNh) is planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by st0nebraker** [comment]
User journey through keyboard, screenreader, and 320px viewport all look good at this time. No perceivable issues.

**Comment by bretthayes** [comment]
I noticed this is marked as done but isn't closed. Just wondering if this is still WIP? @st0nebraker

**Comment by st0nebraker** [comment]
Ah thanks, closing now. Yes it's done 😄
